qgispluginci/release.py

```diff
@@ -393,18 +393,14 @@ def create_plugin_repo(
         "__REPO_URL__": parameters.repository_url,
     }
     if not plugin_repo_url:
-        download_url = "https://github.com/{org}/{repo}/releases/download/{tag}/{pluginzip}".format(
-            org=replace_dict["__ORG__"],
-            repo=replace_dict["__REPO__"],
-            tag=replace_dict["__RELEASE_TAG__"],
-            pluginzip=replace_dict["__PLUGINZIP__"],
+        download_url = (
+            f"https://github.com/{replace_dict['__ORG__']}/{replace_dict['__REPO__']}/"
+            f"releases/download/"
+            f"{replace_dict['__RELEASE_TAG__']}/{replace_dict['__PLUGINZIP__']}"
```

Comment:
I find the readability worse here.
What about create the vars and use `download_url = f"https://github.com/{org}/{repo}/releases/download/{tag}/{pluginzip}"`